src/router/routes/live/events.ts

```diff
@@ -170,6 +170,9 @@ export class EventsController extends LiveController {
     if (!request.body.eventType) {
       return Util.standardErrorHandler(new HttpError('Event type must be provided', 400), next);
     }
+    if (isNaN(Number(request.body.eventLocation))) {
```

Comment:
I chose Number() instead of parseInt because if the input is "123ss" parseInt returns 123 while Number returns NaN